packages/clarity-js/src/data/metadata.ts

```diff
@@ -252,18 +252,19 @@ function processCallback(upgrade: BooleanFlag, consentUpdate: boolean = false):
   }
 }
 
-function track(u: User, consent: BooleanFlag = null): void {
```

Comment:
Rename to analytics to make it unambiguous. Consent own its own is too ambiguous.